### PR TITLE
Add Panda Auth to require auth for main endpoints

### PR DESCRIPTION
**Pull request by paulmr**
and test with builtin keys so S3 access is not a dependency for testing

### CODE REVIEW

File: app/controllers/MainApp.scala [hunk 1]
```diff
@@ -1,31 +1,44 @@
...
+                         val wsClient: WSClient,
+                         val conf: Configuration,
+                         val authActions: AuthActions)
+    extends AtomController {
 
-  def index = Action {
+  import authActions.{ AuthAction, processGoogleCallback }
+
+  def healthcheck = Action {
+    println(com.gu.pandomainauth.PublicSettings.bucketName)
```
Comment (jamesgorrie) [inline app/controllers/MainApp.scala]: 👀

**Comment by jamesgorrie** [comment]
Am I being silly - this will let you login with AWS creds and Panda / Google auth?

**Comment by paulmr** [comment]
@jamesgorrie , hmm, how do you mean? I am not sure I understand.

It *should* only let you login with Panda/Google Auth, but I may have made a mistake ... ?

**Comment by jamesgorrie** [comment]
No - sorry, I misread. It's getting the panda aws credentials.
Looks good from a tools perspective - it'd be good to have a small rundown for clarity too.

**Comment by jamesgorrie** [comment]
👍

**Pull request merged**